automated-comments/csharp/two-fer/use_null_coalescing_operator_not_is_null_or_empty.md

```diff
@@ -0,0 +1 @@
+You are currently using `string.IsNullOrEmpty` to determine if the default name should be used. As the test suite only tests for `null` values and not empty values, you could try using the [null-coalescing operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator) to simplify your code.
```

Comment:
@mikedamay said:

"I would not wish to discourage, in any way, the mentee from thinking beyond the tests so I would say “You are currently using string.IsNullOrEmpty to determine if the default name should be used. For variety, you could try using the null-coalescing operator to simplify your code and consider the difference in behaviour.”.

I know that is not a great alternative but I believe that it is important to discourage coding to the tests."

Reply:
"as an alternative" sounds good.